### PR TITLE
DOCSP-41807 Release notes, redirects, source constant updates for v1.26

### PR DESCRIPTION
**Pull request by kyuan-mongodb**
- [DOCSP-41807](https://jira.mongodb.org/browse/DOCSP-41807)
- [STAGING](https://preview-mongodbkyuanmongodb.gatsbyjs.io/atlas-cli/DOCSP-41807/atlas-cli-changelog/#atlas-cli-1.26.0)
- [LATEST BUILD LOG](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66aa880c824f175afd7b5661)

### Self-Review Checklist

- [ ] Check that the submodule pulled in the right changes (if applicable).
- [ ] [Define](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) taxonomy [values](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) at top of page.
- [ ] Add genre facets (tutorial or reference), as in this [example PR](https://github.com/10gen/cloud-docs/pull/5042).
- [ ] Add programmingLanguage (if necessary).
- [ ] Add meta keywords (if necessary).
- [x] Resolve any new warnings or errors in the build.
- [x] Proofread for spelling and grammatical errors.
- [x] Check staging for rendering issues.
- [x] Confirm links are working.

### CODE REVIEW

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: source/atlas-cli-changelog.txt [hunk 1]
```diff
@@ -14,6 +14,23 @@
...
+
+- Fixes a local deployment error message when you don't have Docker
+  installed on MacOS and Windows. 
+- Fixes the issue with connecting to local deployment using {+mongosh+}
+  and :compass:`Compass </>` on Windows.
+- Updates the :ref:`atlas-deployments-setup` command to always use the
+  latest image when creating a new local deployment.
+- Fixes a bug in the :ref:`atlas-deployments-setup` command where the
+  {+atlas-cli+} deletes an existing local deployment when you run this
+  command twice for the same deployment.
```
Comment (andreaangiolillo) [inline source/atlas-cli-changelog.txt:32]: CC @jeroenvervaeke to double check if something is missing.

Comment (jeroenvervaeke) [reply source/atlas-cli-changelog.txt:32]: LGTM

**Review by jeroenvervaeke — APPROVED** [review]

File: source/atlas-cli-changelog.txt [hunk 2]
```diff
@@ -14,6 +14,23 @@
...
+.. _atlas-cli-1.26.0:
+
+{+atlas-cli+} 1.26.0
+--------------------
+
+Released 25 July 2024
+
+- Fixes a local deployment error message when you don't have Docker
+  installed on MacOS and Windows. 
+- Fixes the issue with connecting to local deployment using {+mongosh+}
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:26]: Nit
```suggestion
- Fixes an issue when connecting to a local deployment using {+mongosh+}
```

**Review by sarahsimpers — APPROVED** [review]
LGTM with one edit

**Pull request merged**